laws/src/main/scala/algebra/laws/package.scala

```diff
@@ -0,0 +1,47 @@
+package algebra
+
+import algebra.ring.AdditiveMonoid
+
+import org.typelevel.discipline.Predicate
+
+import org.scalacheck._
+import org.scalacheck.util.Pretty
+import Prop.{False, Proof, Result}
+
+package object laws {
+
+  implicit def PredicateFromMonoid[A](implicit ev: Eq[A], A: AdditiveMonoid[A]): Predicate[A] =
+    new Predicate[A] {
+      def apply(a: A) = ev.neqv(a, A.zero)
+    }
+
+  lazy val proved = Prop(Result(status = Proof))
+
+  lazy val falsified = Prop(Result(status = False))
+
+  object Ops {
+    def run[A](sym: String)(lhs: A, rhs: A)(f: (A, A) => Boolean): Prop =
```

Comment:
Why is this wrapped in an object? Couldn't this be just a regular function? Sorry I'm late to the party :-)